### DISCUSSION TITLE
Generate adversarial examples for mixed data types and pipelines

### DISCUSSION THREAD

**Discussion by shreyakhandelwal07** [Trusted-AI/adversarial-robustness-toolbox#1433] [Q&A]
How can I use ART to generate adversarial examples for mixed (categorical and continuous) data types? I am using sklearn pipelines for data transformation and classification.

**Comment by beat-buesser** [comment]
Hi @shreyakhandelwal07 It depends, it's not always easily possible. Would you have a representative data example that you can share? It does not have to be your actual data.
